### DISCUSSION TITLE
APCP_06 not defined in scope

### DISCUSSION THREAD

**Discussion by craigr3** [dtcenter/METplus#1834] [Configuration] [answered]
I am getting the subject error when I try to verify a summed precipitation netcdf file.  The precip was summed using PCPCombine add in METPLUS.  The NETCDF header is 
![netcdf header](https://user-images.githubusercontent.com/35153906/192065903-d502543a-5cd4-45f8-ad03-3921dbcfa3b2.png)

The config_file snap shot is : 
![config_file](https://user-images.githubusercontent.com/35153906/192065933-22648468-7334-4397-9f82-24712ede5302.png)

If I do a plot_data_plane on the netcdf file using 'name="APCP_06"; level="A6";  ' , it plots it just fine.  But if I run point_stat on the file I get the following error: 
![error](https://user-images.githubusercontent.com/35153906/192066212-05a67267-70e8-40be-b3c3-8b39bf55242d.png)

Not sure what to try next.  This is MET 10.0.1

**Comment by craigr3** [comment, marked as answer]
Found the problem - It wasn't a MET issue.

**Reply by JohnHalleyGotway** [reply]
Bob, great, thanks for letting us know. I'll mark this one as being done.
